src/App.js

```diff
@@ -1,24 +1,18 @@
-import React, {
-  Component
-} from 'react';
-// import logo from './logo.svg';
+import React, { Component } from 'react';
 import './styles/Main.scss';
 import TrailList from './TrailList.js';
-// import Header from './Header.js';
-// import Search from './Search.js';
 import LocationDisplay from './LocationDisplay.js';
 import LandingScreen from './LandingScreen.js';
 import Controls from './Controls.js';
 
-
-
 class App extends Component {
   constructor() {
     super();
     this.state = {
       landingScreen: true,
       parkData: [],
       trailData: [],
+      filteredTrails: [],
```

Comment:
I like what you did here with getting rid of the fetchTrails method. It was duplicate code getting the same data as our compoundDidMount. Looks like that was the issue with our cards not filtering correctly.